test/FBeetsRevenueSharer.test.ts

```diff
@@ -123,6 +125,37 @@ describe("BeetsBar", function () {
     expect(await beets.balanceOf(bob.address)).to.be.closeTo(userInfo.rewardDebt.div(2), 200000)
   })
 
+  it("change fBeets locker share to 75%", async () => {
```

Comment:
i always try to formulate tests based on use cases, so for me the use case of this test would be `distributes correct amount to lockers after changing the share` or something like that.

Reply:
Better?

Reply:
beauty :pray: